db/database.go

```diff
@@ -2567,3 +2568,31 @@ func (db *Database) DataStoreNames() base.ScopeAndCollectionNames {
 	}
 	return names
 }
+
+// GetCollectionIDs will return all collection IDs for all collections configured on the database
+func (db *DatabaseContext) GetCollectionIDs() []uint32 {
+	collIDs := make([]uint32, 0)
+	for id, _ := range db.CollectionByID {
+		collIDs = append(collIDs, id)
+	}
+	return collIDs
+}
```

Comment:
from `exp/x/maps` package, and then a bit different in go 1.23 std library maps with iterators: `slices.Collect(maps.Keys(db.CollectionByID)`

```suggestion
	return maps.Keys(db.CollectionByID)
}
```